common/decklist.cpp

```diff
@@ -466,66 +466,111 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
 {
     cleanList();
+    QVector<QString> inputs; // QTextStream -> QVector
 
-    bool inSideboard = false, isSideboard = false;
+    bool priorEntryIsBlank = true, isAtBeginning = true;
```

Comment:
Its not inefficient, though it can be less readable, as one line is doing more than one thing. You gain saving a line over readability. I prefer readability.